src/fstab-generator/fstab-generator.c

```diff
@@ -813,7 +813,9 @@ static bool sysfs_check(void) {
         return cached;
 }
 
-static int add_sysusr_sysroot_usr_bind_mount(const char *source) {
+static int add_sysusr_sysroot_usr_bind_mount(const char *source, bool validatefs) {
```

Comment:
why use a "bool" here btw? given we otherwise use a flags field for this i think we should stay true to that theme and also use the flags type here, even if we only need to ever use a single bit of it...